README.md

```diff
@@ -167,17 +173,27 @@ df.tensorflow.to_tfr(
 ```
 
 ### Supported types
+TFRecorder's schema system supports several types, all listed below. You can use
+these types by referencing them in the schema map. Each type informs TFRecorder how 
+to treat your DataFrame columns.  For example, the schema mapping 
+`my_split_key: schema.split_key` tells TFRecorder to treat the column `my_split_key` as
+type `schema.split_key` and create dataset splits based on it's contents. 
 
 #### schema.image_uri 
-* Specifies the path to an image. When specifying, TFRecorder
-will load the specified image and store the image as a tf.string
+* Specifies the path to an image. When specified, TFRecorder
+will load the specified image and store the image as a [tf.string](https://www.tensorflow.org/tutorials/load_data/unicode)
```

Comment:
Maybe good to mention that image is base64 encoded.

Reply:
Good idea. I added that and further descriptions on keys.